android-amazongc/src/de/golfgl/gdxgamesvcs/GameCircleClient.java

```diff
@@ -100,7 +101,7 @@ protected void agcServiceNotReady(AmazonGamesStatus amazonGamesStatus) {
         isConnectionPending = false;
         isConnected = false;
         if (gsListener != null) {
-            gsListener.gsOnSessionInactive();
+            gsListener.gsOnSessionInactive(null);
```

Comment:
Why `null`? Wasn't this a successfull operation? Same for all over calls with null.

Reply:
Went through each callback and added correct response codes